packages/blitz-next/src/index-server.ts

```diff
@@ -50,6 +53,7 @@ export type NextApiHandler<TResult> = (
 type SetupBlitzOptions = {
   plugins: BlitzServerPlugin<RequestMiddleware, Ctx>[]
   onError?: (err: Error) => void
+  log: {logger: ReturnType<typeof BlitzLogger>; logLevel?: BlitzLogLevel}
```

Comment:
can we simplify this API? Because logLevel will only work with our default logger, so doesn't make sense to have it here I don't think. They can just set `minLevel` in the default logger options

```suggestion
  logger: ReturnType<typeof BlitzLogger>;
```